http/_bench.pony

```diff
@@ -0,0 +1,183 @@
+use "ponybench"
+use "buffered"
+use "debug"
+
+class PrivateBenchmarks is BenchmarkList
+  fun tag benchmarks(bench: PonyBench tag) =>
+    Debug("benching private benchmarks")
+    bench(_SimpleGetRequestBenchmark)
+    bench(_FormSubmissionRequestBenchmark)
+    bench(_SplitFormSubmissionRequestBenchmark)
+//    bench(_MultipartFileUploadBenchmark)
```

Comment:
Is there a reason why you are committing commented out code?

Reply:
I just wanted to share the basic setup already. Those benchmarks are going to be implemented as part of this PR. If i eouldnt comment them, they would timeout or run very very long, as they do not complete.

Reply:
i added the benchmarks that were previously commented out.